languagetool-language-modules/pt/src/main/resources/org/languagetool/resource/pt/disambiguation.xml

```diff
@@ -3466,6 +3466,20 @@
         <disambig action="ignore_spelling"/>
     </rule>
 
+    <rule id="HEXADECIMAL_NOTATION">
```

Comment:
The 'name' attribute is missing. I think it is required in some tests.

Reply:
This is a rule within a rulegroup, though, and the rulegroup has both a `name` and an `id` attribute. Should be fine..?

Reply:
Probably yes. If the tests pass, it is fine.